compiler_tester/src/vm/eravm/system_context.rs

```diff
@@ -128,11 +128,11 @@ impl SystemContext {
 
         let block_number = match target {
```

Comment:
Btw, any reason to have different blocks behaviour for EraVM and EVM? We are running the solidity semanticTests against EraVM as well, so maybe it makes sense to have the same context behaviour

Reply:
I realized the reason I had to have different behaviour here was because I was incrementing block number before and not after modifying storage. On this commit [cb080d6](https://github.com/matter-labs/era-compiler-tester/pull/82/commits/cb080d6725c2f29d3f6f9ab39b446973d151e06f) I reworked it so now block behaviours are unified again on the system context. Let me know what you think

Reply:
@AntonD3 they are located in different repos.
Updating a single context variable won't make them more compatible.